### PR TITLE
[task-03] [bug] [Organize First File]

### PR DESCRIPTION
**Pull request by samnoonabrar**
## What Happened

[task-03]Bug] [Organize First File]
## Insights

-- Added A New File
Proof
image

### CODE REVIEW

**Comment by sayanta28** [comment]
Update Pull Title.

**Comment by samnoonabrar** [comment]
Done.

> Update Pull Title.

File: new.txt [hunk 1]
```diff
@@ -1 +1 @@
-ansdnaklsjd;ald;lkas
\ No newline at end of file
+Hello world
```
Comment (MahinHossainMunna) [inline new.txt:1]: Nice Work

**Review by MahinHossainMunna — APPROVED** [review]

**Review by sayanta28 — APPROVED** [review]
Nice Work. I Like it.

**Comment by FForhad** [comment]
Appreciating work.

**Pull request merged**